cavityFaultDisplay.py

```diff
@@ -34,7 +35,7 @@ def __init__(self, parent=None, args=None, macros=None):
             codeLabel.setSizePolicy(QSizePolicy.Maximum, QSizePolicy.Maximum)
 
             nameLabel = QLabel()
-            nameLabel.setText(fault.name)
+            nameLabel.setText(fault.shortDescription)
```

Comment:
I would also probably rename nameLabel

Reply:
Good catch! Changing it now for consistency